site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -195,6 +195,15 @@ one of the following supported types:
     - `LocalDate`, `LocalDateTime` and `LocalTime`,
     - `OffsetDateTime` and `OffsetTime`
     - `ZonedDateTime`, `ZoneId` and `ZoneOffset`
+- Custom types with either of the following public static methods taking a single `String` parameter. Note that
+  these are scanned for in the order below the first matching method being used,
```

Comment:
`the below order, the first`?

Reply:
Maybe better splitting this sentence into multiple ones for readability?

Reply:
Agreed.